http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=531986

the current version fails to build on GNU/kFreeBSD.
It needs small tweak. The work with shared libraries
is not determined by kernel, but by glibc.

Petr Salinger <Petr.Salinger@seznam.cz>

--- a/src/g_initialize.c	2007-09-05 00:10:37.000000000 +1000
+++ b/src/g_initialize.c	2009-06-06 18:37:52.000000000 +1000
@@ -44,7 +44,7 @@
 static void solaris_initialize (void);
 #endif /* USE_SOLARIS_SHARED_LIBRARIES */
 
-#ifdef __linux__
+#if defined(__linux__) || defined(__GLIBC__)
 #define USE_LINUX_SHARED_LIBRARIES
 #endif
 
